building-web-forms/learn-more-forms/README.md

```diff
@@ -0,0 +1,18 @@
+# Text
+
+> This is optional reading. In this target, we'll link you to external documentation that you can look at if you want to. Do that only if you are curious and have the time to spare. Otherwise, you can safely skip this chapter.
```

Comment:
```suggestion
> This is optional reading. In this target, we'll link you to external documentations that goes in depth about `form` element.  You can safely skip this chapter.
```

Reply:
I am reusing the same text from existing WD101 for this. I assume it would be better as it was reviewed and included earlier.